plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizard.java

```diff
@@ -59,7 +60,9 @@ public void addPages() {
 	public boolean performFinish() {
 		this.serverSettingsWizardPage.getModel().updateServer();
 		try {
-			this.createdServer = this.serverSettingsWizardPage.getModel().getServer().save(true, new NullProgressMonitor());
+			IServerWorkingCopy wc = this.serverSettingsWizardPage.getModel().getServer();
+			wc.setAttribute(OpenShiftServerUtils.SERVER_START_ON_CREATION, true);
```

Comment:
I see this being made in the wizard and the page with twice the same code. Why not do it upon saving the server in the model?